Uplift/Views/Pickup Games/ProfileView.swift

```diff
@@ -0,0 +1,226 @@
+    private let gamesSections: [GamesListHeaderSections] = [.myGames, .joinedGames, .pastGames]
+    private let profilePictureSize: CGFloat = 60
+
+    // MARK: - Data
+    private var myGames: [Post] = []
+    private var joinedGames: [Post] = []
+    private var pastGames: [Post] = []
+
+    init(frame: CGRect, myGames: [Post] = [], joinedGames: [Post] = [], pastGames: [Post] = []) {
+
+        let layout = UICollectionViewFlowLayout()
+        layout.minimumLineSpacing = 24
+        collectionView = UICollectionView(frame: .zero, collectionViewLayout: layout)
+
+        super.init(frame: frame)
+
+        self.myGames = myGames
+        self.joinedGames = joinedGames
+        self.pastGames = pastGames
+
+        backgroundColor = .white
+
+        profilePicture.image = UIImage(named: ImageNames.profilePicDemo)
+        profilePicture.layer.cornerRadius = profilePictureSize
+        addSubview(profilePicture)
+
+        nameLabel.text = "Zain Khoja"
+        nameLabel.font = ._24MontserratBold
+        nameLabel.textAlignment = .center
+        nameLabel.textColor = .black
+        addSubview(nameLabel)
+
+        gamesPlayedLabel.text = "27 games played"
+        gamesPlayedLabel.font = ._14MontserratMedium
+        gamesPlayedLabel.textAlignment = .center
+        gamesPlayedLabel.textColor = .gray04
+        addSubview(gamesPlayedLabel)
+
+        collectionView.register(PickupGameCell.self, forCellWithReuseIdentifier: Identifiers.pickupGameCell)
+        collectionView.register(GamesListHeaderView.self, forSupplementaryViewOfKind: UICollectionView.elementKindSectionHeader, withReuseIdentifier: Identifiers.gamesListHeaderView)
+        collectionView.contentInset = UIEdgeInsets(top: collectionViewTopInset, left: 0, bottom: collectionViewBottomInset, right: 0)
+        collectionView.backgroundColor = .clear
+        collectionView.delegate = self
+        collectionView.dataSource = self
+        collectionView.showsVerticalScrollIndicator = false
+        addSubview(collectionView)
+
+        gradientViewTop.backgroundColor = .white
+        addSubview(gradientViewTop)
+
+        gradientViewBottom.backgroundColor = .white
+        addSubview(gradientViewBottom)
+
+        setupConstraints()
+    }
+
+    func setupGradients() {
+        let gradientTop = CAGradientLayer()
+        gradientTop.frame = gradientViewTop.bounds
+        gradientTop.colors = [UIColor.white.cgColor, UIColor.white.withAlphaComponent(0.5).cgColor, UIColor.clear.cgColor]
+        gradientTop.locations = [0.1, 0.5, 1]
+//        gradientViewTop.layer.addSublayer(gradientTop)
+        gradientViewTop.layer.mask = gradientTop
+
+//        print(gradientViewBottom)
+        let gradientBottom = CAGradientLayer()
+        gradientBottom.frame = gradientViewBottom.bounds
+        gradientBottom.colors = [UIColor.clear.cgColor, UIColor.white.withAlphaComponent(0.5).cgColor, UIColor.white.cgColor]
+        gradientBottom.locations = [0.1, 0.5, 1]
```

Comment:
nits but can we:
1) put the locations inside a constant since they're the same
2) separate the colors across multiple lines since its kind of hard to read
```
gradientTop.colors = [
  UIColor.white.cgColor, 
  UIColor.white.withAlphaComponent(0.5).cgColor, 
  UIColor.clear.cgColor
]
```